lib/msal-common/src/utils/Constants.ts

```diff
@@ -401,3 +401,21 @@ export enum JsonTypes {
     Jwt = "JWT",
     Jwk = "JWK"
 }
+
+export enum SilentTokenRetrievalStrategy {
+    // Look in cache, then use existing RT, then renew RT (existing default behavior)
+    CacheAndNetwork = 0, // 0 is falsy, is equivalent to not passing in a SilentTokenRetrievalStrategy
+    // Only look in cache, and dont go to network (either there's a cached AT, or throw error)
+    CacheOnly = 1,
+    /*
+     * look in cache, if not in cache then use existing RT to get AT
+     * if existing RT is not valid, do not attempt to silently renew it
+     */
+    CacheOrExistingRefreshToken = 2,
+    // Only go to network, use existing RT but don't renew it
+    NetworkWithExistingRefreshTokenOnly = 3,
+    // Only go to network, use existing RT and renew it (existing forceRefresh flag equals true)
+    NetworkWithRefreshToken = 4,
+    // Only go to network, don't use existing RT, get a new one
```

Comment:
Can you clean up these descriptions? They will be shown to developers in their IDE.

```suggestion
    // acquireTokenSilent will only go to network, and will not use cached access token or refresh token
```